doc/rtd/topics/debugging.rst

```diff
@@ -163,3 +163,92 @@ commandline:
 
 Inspect cloud-init.log for output of what operations were performed as a
 result.
+
+.. _proposed_sru_testing:
+
+Stable Release Updates (SRU) testing for cloud-init
+==============================================================
+Once an Ubuntu release has been published, updates to it must follow a special
+procedure called a "stable release update" or `SRU`_.
+
+The cloud-init project has an exception process it follows when validating
+cloud-init for an SRU listed at `CloudinitUpdates`_.
+
+Generally SRU tests of cloud-init perform the following:
```

Comment:
If we have this as "SRU tests", then I think this list should describe what an individual test that a reader would perform does. In which case, "works properly on various platforms or Ubuntu series" isn't quite right: an individual test will test a single platform/series combination.

Alternatively, we could change this to refer to "SRU testing" more generally, in which case I think the current wording is fine (and my comment about performance below is perhaps unnecessary).

Does this make sense? What do you think?

Reply:
+1 changed to individual SRU test.